libcloud/test/compute/test_openstack.py

```diff
@@ -1697,6 +1679,38 @@ def test_ex_accept_image_member(self):
         self.assertEqual(image_member.extra['updated'], '2018-03-02T14:20:37Z')
         self.assertEqual(image_member.extra['schema'], '/v2/schemas/member')
 
+    def test_ex_list_networks(self):
+        networks = self.driver.ex_list_networks()
+        network = networks[0]
+
+        self.assertEqual(len(networks), 2)
+        self.assertEqual(network.name, 'net1')
```

Comment:
noticed that you left out the `self.assertEqual(network.cidr` that was here before, doesn't really matter but not sure if that was intended

Reply:
I remove it as it is allways None in real cases so, I think that it have no sense to set a value in the tests that will never appear. But If you think that is interesting we can add it again,